lib/authentication/password/link_sent.dart

```diff
@@ -80,10 +85,7 @@ class _LinkSent extends State<LinkSent> {
       width: screenWidth,
       child: Column(
         children: <Widget>[
-          SvgPicture.asset(
-            'images/diverse_ano.svg',
-            height: screenHeight * 0.11,
-          ),
+          Image.asset('images/sign_up/ano_hands_side.png', height: 100),
```

Comment:
Could we adjust the height of the image based on the `screenHeight` as well? I find that it's more responsive that way, specially in smaller screens.

Reply:
I'll update other instances too!

Reply:
Updated all but the one in `new_matches`. I don't know how to get messages showing up on the page, and it shows only the global header by default and nothing else.

Reply:
Thank you!!! ❤️